tasks/uninstall.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Remove the APPUiO Pruner
+  with_items:
+    - CronJob/prune-builds
+    - CronJob/prune-deployments
+    - CronJob/prune-images
+    - ServiceAccount/pruner
+    - ClusterRoleBinding/appuio-pruner
+    - ClusterRoleBinding/appuio-image-pruner
+    - ClusterRole/appuio-pruner
+  command: >-
+    oc -n {{ appuio_pruner_namespace | quote }}
+    delete --ignore-not-found {{ item | quote }}
```

Comment:
Proposal: Tag all objects with a label/value combination, then use `--label foo=bar` to delete them.

Reply:
Ah, wanted to add labels either way but that task got lost on the way